README.md

```diff
@@ -19,10 +19,10 @@ In order to use Salesforce SSO authentication, you **must** create a connected a
 1. On the top right, click the "New Connected App" button.
 1. Enter an app name, API name, and contact email of your choice.
 1. Click the "Enable OAuth Settings" toggle.
-1. Enter a Callback URL to allow Salesforce to redirect users back to your Pardot plugin settings page. The URL should look similar to: `https://[YourWordpressDomainHere]/wp-admin/options-general.php?page=pardot`.
-1. Add "Access Pardot Services (pardot_api)" and "Perform requests on your behalf at any time (refresh_token, offline_access)" to your selected OAuth scopes.
+1. Enter a Callback URL to allow Salesforce to redirect users back to your Account Engagement plugin settings page. The URL should look similar to: `https://[YourWordpressDomainHere]/wp-admin/options-general.php?page=pardot`.
+1. Add "Access Account Engagement Services (pardot_api)" and "Perform requests on your behalf at any time (refresh_token, offline_access)" to your selected OAuth scopes.
```

Comment:
Unfortunately our Oauth scope is still called Manage Pardot Services (pardot_api), I couldn't find it in Pardot codebase/package, likely it's in Core, we should keep it till we change it.

Reply:
@nsweeneypi  there's a tricky part for rebrand. Our plugin itself is called Pardot as url :https://wordpress.org/plugins/pardot/  I don't find direct option to change it to Account Engagement. Maybe there's a admin page. Would keep name Pardot look really bad?

Reply:
@pengchensfdc, I'm okay keeping the URL set as _/pardot_, but you're not suggesting we still call it the Pardot Wordpress App, are you? If so, I say we still call it _Account Engagement_ even if the URL still says _pardot_, just like pardot.com.

Reply:
I'm not too concern with url but more the plugin name itself instead. Like you can check the url, the plugin is called Pardot, the same thing as we search the plugin in plugin store inside wordpress. We already changed the plugin name at: trunk/pardot.php, I think I can verify if it change the way / capatability to existing implementation when I test it.

Reply:
@pengchensfdc, if we ever want the new name to stick, we need to change it across the board, even if the code stays as _pardot_. 

We need to call this the _Account Engagement Wordpress Plugin_ because the app is now called _Account Engagement_, not _Pardot_.